requirements.txt

```diff
@@ -61,14 +61,14 @@ requests==2.25.1
 requests-oauthlib==1.3.0
 six==1.15.0
 soupsieve==2.1
-sqlparse==0.4.2
+sqlparse==0.4.1
 swapper==1.1.2.post1
-urllib3==1.26.8
+urllib3==1.26.2
 vine==5.0.0
 wcwidth==0.2.5
 webencodings==0.5.1
 whitenoise==5.2.0
 xhtml2pdf==0.2.5
 xlrd==2.0.1
 XlsxWriter==1.3.7
-xlwt==1.3.0
+xlwt==1.3.0
```

Comment:
The changes in the requirements file are not acceptable. Kindly remove these changes from the file.

Reply:
@Smeetiiitdmj These changes are not removed yet. Kindly remove these changes.

Reply:
Kindly check now, I have committed